data/sql/updates/pending_db_world/update-jinzallah-spawn-posis.sql

```diff
@@ -0,0 +1,2 @@
+--
+UPDATE `creature` SET `position_x` = -7195.0405, `position_y` = -2979.6877, `position_z` = 9.052029, `orientation` = 5.340707302093505859 WHERE `guid` = 51829 AND `id1` = 8200;
```

Comment:
Actually, you can add the VerifiedBuild as well 😛 
Just to differentiate that this particular spawn is sniffed